feature_layers/feature-collection-layer-from-portal/README.md

```diff
@@ -0,0 +1,34 @@
+# Feature Collection Layer From Portal
+
+This sample demonstrates opening a feature collection saved as a portal item
```

Comment:
We're trying to keep the readme as close to the common desing as possible, so I'd suggest sticking with the same short description as we have there:

```suggestion
Create a feature collection layer from a portal item.
```